person.rb

```diff
@@ -4,14 +4,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] As already mentioned above, the rental variable can equally be modified. Let's kindly make it an `attr_accessor`